docs/dev/setup.rst

```diff
@@ -257,6 +258,126 @@ You can stop the service using ``sudo service artemis stop`` and restart it usin
+              gitlab_rails['gitlab_default_projects_features_container_registry'] = false
+              gitlab_rails['backup_keep_time'] = 604800
+        ports:
+          - "${GITLAB_SSH_PORT}:22"
+          - "${GITLAB_HTTP_PORT}:80"
+        networks:
+          - artemis-net
+
+      jenkins:
+        image: jenkins/jenkins:lts
+        restart: unless-stopped
+        user: root
+        volumes:
+          # - $BACKUP_DIR/jenkins:/var/jenkins_backup # Optional but useful
+          - ./data/jenkins/home:/var/jenkins_home
+          - /var/run/docker.sock:/var/run/docker.sock
+          - /usr/bin/docker:/usr/bin/docker:ro
+        ports:
+          - "${JENKINS_HTTP_PORT}:8080"
+          - "50000:50000"
+        networks:
+          - artemis-net
+
+      artemis:
+        image: ghcr.io/ls1intum/artemis:${ARTEMIS_VERSION:-latest}
+        restart: unless-stopped
+        depends_on:
+          - artemis-db
+          - gitlab
+          - jenkins
+        volumes:
+          - ./data/artemis/config:/opt/artemis/config
+          - ./data/artemis/data:/opt/artemis/data
+          - ./branding:/opt/artemis/public/content:ro
+        environment:
+          - spring.profiles.active=prod,jenkins,gitlab,artemis,scheduling
+        ports:
+          - "${ARTEMIS_HTTP_PORT:-8080}:8080"
+        networks:
+          - artemis-net
+
+      artemis-db:
+        image: mysql:8.0.23
+        restart: unless-stopped
+        volumes:
+          - ./data/artemis-db:/var/lib/mysql
+        environment:
+          - MYSQL_ALLOW_EMPTY_PASSWORD=yes
+          - MYSQL_DATABASE=Artemis
+        command: mysqld --lower_case_table_names=1 --skip-ssl --character_set_server=utf8mb4 --collation-server=utf8mb4_unicode_ci --explicit_defaults_for_timestamp
+        networks:
+          - artemis-net
+        cap_add:
+          - SYS_NICE
+
+    networks:
+      artemis-net:
+        ipam:
+          driver: default
+          config:
+            - subnet: 10.1.0.0/16 # Arbitrary, but set this to the IPs your department defines for local docker networks
+
+
+You can find the latest Dockerfile with additional information `here <https://github.com/ls1intum/Artemis/blob/develop/src/main/docker/Dockerfile>`__.
+
+
+* The Dockerfile defines three Docker volumes
+
+    * ``/opt/artemis/config``: This will be used to store the configuration of Artemis in YAML files. If this directory is empty, the default configuration of Artemis will be copied upon container start.
+    * ``/opt/artemis/data``: This directory should be used for any data (e.g., local clone of repositories). Therefore, configure Artemis to store this files into this directory. In order to do that, you have to change some properties in configuration files (e.g., `artemis.repo-clone-path`, `artemis.repo-download-clone-path`, `artemis.course-archives-path`, `artemis.submission-export-path`, and `artemis.file-upload-path`). Otherwise you'll get permission failures.
```

Comment:
```suggestion
    * ``/opt/artemis/data``: This directory should be used for any data (e.g., local clone of repositories). Therefore, configure Artemis to store this files into this directory. In order to do that, you have to change some properties in configuration files (i.e., ``artemis.repo-clone-path``, ``artemis.repo-download-clone-path``, ``artemis.course-archives-path``, ``artemis.submission-export-path``, and ``artemis.file-upload-path``). Otherwise you'll get permission failures.
```

Reply:
@b-fein Whats the difference between `` and ` in RST files. I'm not using them often :)

Reply:
Only in Markdown a single backtick is enough to make it `monospaced`, but in RST files you need two for that. I think the single backticks are reserved as markers for document processors, e.g. for the way links and their displayed text should be formatted.